core-bundle/src/Resources/contao/dca/tl_templates.php

```diff
@@ -266,12 +267,6 @@ public function addNewTemplate()
 		if (Contao\Input::post('FORM_SUBMIT') == 'tl_create_template')
 		{
 			$strOriginal = Contao\Input::post('original', true);
```

Comment:
I guess we can move this line to line 285 now, can‘t we? Right below `$blnFound = false;`.

Reply:
Agree.